### ISSUE TITLE
[Drilldown Menu] Voiceover accessibility issue 

### ISSUE THREAD

**Issue by karissademi** [zurb/foundation-sites#9271]
**How can we reproduce this bug?**
1. Open the [drilldown menu](http://foundation.zurb.com/sites/docs/drilldown-menu.html) example on a mobile device. I've tested on a Nexus 5x and iPhone 5. 
2. Open up your availble voiceover mode (TalkBack on Android, VoiceOver on iOS)
3. Attempt to navigate the drilldown in this mode. 

**What did you expect to happen?**
A user should be able to tap a menu item once to have the label read and a double tap for the item to drill to another submenu. 

**What happened instead?**
If you navigate to a submenu, the menu gets stuck at the `<ul>`, and behaves erratically from this point on.

Inspecting the page reveals that the value of `aria-hidden="false"` never gets set to true when a drilldown menu opens. 

I've corrected this issue for an implementation that requires it by updating the value of `aria-hidden` for items that have the `is-active` class when `open.zf.drilldown` or `hide.zf.drilldown` are fired.

**Comment by Owlbertz** [comment]
Hi @karissademi thank you for reporting this. If you already have a fix for that, do you mind creating a PR?

**Comment by karissademi** [comment]
Hi @Owlbertz, I didn't make the change to the foundation js directly, but I can probably whip that together this weekend.

**Comment by DanielRuf** [comment]
Hi @karissademi,

do you still plan to provide the PR and is the ossue still in the latest release?

**Comment by DanielRuf** [comment]
Hi @karissademi, thanks for the feedback.
